### ISSUE TITLE
Quadpy is now asking for a sigma license

### ISSUE THREAD

**Issue by PGadoury** [sigma-py/quadpy#463]
Upon importing after the latest update, quadpy is requiring a license key for sigma, and throws a license error if the license is not provided.

I'm unsure where to get such a license.

Cheers!

**Comment by nschloe** [comment]
When updating quadpy, the message will disappear (for now).

**Comment by PGadoury** [comment]
I've upgraded my quadpy through pip
pip install quadpy --upgrade

Is there something i'm missing?

**Comment by nschloe** [comment]
What version are you running now?

**Comment by mschytt** [comment]
Having the same problem. Upgraded to 0.16.17 and is prompted a Sigma license before the import can complete.

**Comment by PGadoury** [comment]
I'm running quadpy-0.16.17

**Comment by nschloe** [comment]
Looking into it, hang on.

**Comment by nschloe** [comment]
Could you uninstall your quadpy version and force reinstall.

**Comment by mschytt** [comment]
Works as intended!

**Comment by PGadoury** [comment]
uninstall (and a force uninstall of orthopy) followed by force reinstall did the trick, thanks!

**Comment by DanielRudrich** [comment]
Can't get it to work. Could you please list the required versions of `quadpy`, `orthopy`, and `x21` to make it work without requiring a license? I can't really downgrade quadpy as there are only two versions available...

**Comment by nschloe** [comment]
@DanielRudrich The latest versions of everything should do it. Perhaps x21<0.4, but that's a requirement. Have you tried reinstalling everyhing? What versions do you have installed?

**Comment by DanielRudrich** [comment]
quadpy 0.16.19
orthopy 0.8.95
x21 0.3.6

**Comment by nschloe** [comment]
What's your error message?

**Comment by DanielRudrich** [comment]
Just the enter license prompt:

```
Warning: Unable to find valid license in
- environment variable SIGMA_LICENSE_KEY (env var not found), or
- config file /Users/rudrich/Library/Application Support/sigma/sigma.toml (file not found)

Enter new license key for sigma: 
```

**Comment by nschloe** [comment]
What package does it mention before that?

**Comment by DanielRudrich** [comment]
Found the culprit, also had to update the `ndim` package.
Had `ndim 0.1.9`, solved with `0.1.16`

Thanks for your quick response!

Have you already decided on a date to start enforcing valid licenses?

**Comment by nschloe** [comment]
Not entirely sure yet. How far ahead would you like to know?
